modules/internal/DefaultRectangular.chpl

```diff
@@ -868,9 +868,16 @@ module DefaultRectangular {
     }
   
     // only need second version because wrapper record can pass a 1-tuple
-    inline proc dsiAccess(ind: idxType ...1) ref where rank == 1
+    inline proc dsiAccess(ind: idxType ...1) ref
+    where rank == 1
```

Comment:
OK. Looking at it now, having an empty line between each decl helps.

Let's also add an empty line each between the dsiAccess(ind : rank*idxType) decls below.